pkg/proxy/proxy.go

```diff
@@ -111,7 +111,35 @@ func newRedirect(port uint16, source ProxySource, id string) *Redirect {
 	}
 }
 
-func (r *Redirect) updateAccounting(t accesslog.FlowType, verdict accesslog.FlowVerdict) {
+func (r *Redirect) DeriveEndpointInfo(ip net.IP, info *accesslog.EndpointInfo) bool {
+	if ep := endpointmanager.LookupIPv4(addressing.DeriveCiliumIPv4(ip).String()); ep != nil {
+		ep.Mutex.RLock()
+		defer ep.Mutex.RUnlock()
+
+		info.ID = uint64(ep.ID)
+		info.Labels = ep.GetLabels()
+		info.LabelsSHA256 = ep.GetLabelsSHA()
+		info.Identity = uint64(ep.GetIdentity())
+
+		return true
+	}
+
+	return false
+}
+
+// GetObservationPoint returns the observation point at which the redirect is
+// attached to
+func (r *Redirect) GetObservationPoint() accesslog.ObservationPoint {
+	if r.ingress {
+		return accesslog.Ingress
+	}
+
+	return accesslog.Egress
+}
+
+// UpdateAccounting is called for each log record emitted as soon as the
+// verdict is known
+func (r *Redirect) UpdateAccounting(t accesslog.FlowType, verdict accesslog.FlowVerdict) {
```

Comment:
Remove. This doesn't seem to be called anywhere anymore.

Reply:
```
pkg/proxy/logger/logger.go-     lr := LogRecord{
pkg/proxy/logger/logger.go-             LogRecord: accesslog.LogRecord{
pkg/proxy/logger/logger.go-                     Type:              t,
pkg/proxy/logger/logger.go:                     ObservationPoint:  producer.GetObservationPoint(),
pkg/proxy/logger/logger.go-                     IPVersion:         accesslog.VersionIPv4,
pkg/proxy/logger/logger.go-                     TransportProtocol: 6,
pkg/proxy/logger/logger.go-                     Timestamp:         time.Now().UTC().Format(time.RFC3339Nano),
```

Reply:
```
pkg/proxy/logger/logger.go-             lr.Verdict = v
pkg/proxy/logger/logger.go-             lr.Info = info
pkg/proxy/logger/logger.go-
pkg/proxy/logger/logger.go:             lr.producer.UpdateAccounting(lr.Type, lr.Verdict)
pkg/proxy/logger/logger.go-     }
pkg/proxy/logger/logger.go-}
pkg/proxy/logger/logger.go-
```